std_msgs/Header header
ultralytics_ros/BBox2DArray detections
sensor_msgs/Image[] masks
